src/lib/budgetWorkflow.ts

```
@@ -0,0 +1,178 @@
+        .eq("budget_request_id", request.id);
+
+    if (itemsError || !items) return request as BudgetRequest;
+
+    return { ...request, line_items: items } as BudgetRequest;
+}
+
+/**
+ * Fetch all pending budget requests (for the Student Union admin dashboard).
+ */
+export async function fetchPendingBudgetRequests(): Promise<BudgetRequest[]> {
+    const { data, error } = await supabase
```

ESLint

[error] Delete `··`

(prettier/prettier)